presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/TaskProcessors.java

```diff
@@ -34,7 +37,8 @@ private TaskProcessors() {}
 
     public static PairFlatMapFunction<Iterator<SerializedPrestoSparkTaskDescriptor>, Integer, PrestoSparkRow> createTaskProcessor(
             PrestoSparkTaskExecutorFactoryProvider taskExecutorFactoryProvider,
-            CollectionAccumulator<SerializedTaskStats> taskStatsCollector)
+            CollectionAccumulator<SerializedTaskStats> taskStatsCollector,
+            Map<String, Broadcast<List<PrestoSparkSerializedPage>>> broadcastInputs)
```

Comment:
nit: what about `fragementIdToBroadcastInputs`? Same for the other variables in this class.

Reply:
Added a comment instead